packet/models.go

```diff
@@ -221,11 +221,13 @@ type IP struct {
 
 // OperatingSystem holds details for the operating system
 type OperatingSystem struct {
-	Slug     string `json:"slug"`
-	Distro   string `json:"distro"`
-	Version  string `json:"version"`
-	ImageTag string `json:"image_tag"`
-	OsSlug   string `json:"os_slug"`
+	Slug          string `json:"slug"`
+	Distro        string `json:"distro"`
+	Version       string `json:"version"`
+	ImageTag      string `json:"image_tag"`
+	OsSlug        string `json:"os_slug"`
+	Installer     string `json:"installer,omitempty"`
+	InstallerData string `json:"installer_data,omitempty"`
```

Comment:
Yeah, I can change this to a `map[string]interface{}` to keep it generic but enforce a base format. My hope is to keep it generic in case there is a future case for a different generic installer. Granted, that may be unnecessary. Thoughts?

Reply:
Will go ahead and just use `map[string]string` if for some reason we need something else in the future we can handle it then.

Reply:
http://c2.com/xp/YouArentGonnaNeedIt.html
Any chance you would reconsider a proper struct?